tests/pages/firefox_dnt.py

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+from selenium.webdriver.support.select import Select
+from selenium.webdriver.support.ui import WebDriverWait as Wait
+
+from .base import BasePage
+
+
+class FirefoxDNTPage(BasePage):
+
+    _url = '{base_url}/{locale}/firefox/dnt'
+
+    _dnt_status = (By.CSS_SELECTOR, '#dnt-status > h5')
+    _faq_1_header = (By.ID, 'faq-1-header')
+    _faq_1_panel = (By.ID, 'faq-1-panel')
+
+    def dnt_status_is_visible(self):
+        el = self.selenium.find_element(*self._dnt_status)
+        return el.is_displayed()
```

Comment:
I'd rename this to `is_do_not_track_status_visible`. You can also add the `@property` decorator so that you can call this without needing the `()`.